configure.ac

```diff
@@ -182,7 +182,7 @@ AC_SUBST(STRICT_CFLAGS)
 # during 'make distcheck'.
 AC_ARG_ENABLE([gir-doc],
     [AS_HELP_STRING([--enable-gir-doc],
-        [Build GIR documentation for Javascript @<:@default=no@:>@])])
+        [Build GIR documentation for Javascript @<:@default=yes@:>@])])
```

Comment:
Also need to pass the last two parameters to AC_ARG_ENABLE the same way they are passed above for strict-flags (c.f., https://autotools.io/autoconf/arguments.html).